types/src/hiring/index.ts

```diff
@@ -91,23 +91,17 @@ export class InactiveApplicationStage extends JoyStruct<InactiveApplicationStage
 
 export class ActiveApplicationStage extends Null { };
 
+// TODO: Find usages and replace with "JoyEnum-standard"
 export enum ApplicationStageKeys {
```

Comment:
we used the `Keys` suffix for the variants of the JoyEnum.
When I was seeing this type imported in various places it confused me a little, maybe its not the best choice of name, but not sure its worth refactoring now.

Reply:
I was generally following the pattern where:
- `Keys` suffix would be either for `enum` types or `consts` marked with 'as const`, which allows us to access a specific key like `someKey === Keys.Key`
- `Key` suffix would be for string literal unions, ie. `const key: Key = 'SomeKey'`

I think I broke those rules with `WorkingGroupKeys`, which is a string literal.
I could make a quick fix for that.
At some point I think it we could further "normalize" our enums, so that all of them use `JoyEnum` and we can adjust the standards or `JoyEnum` itself to cover all the typical use-cases.

Reply:
Renamed `WorkingGroupKeys` to `WorkingGroupKey` to match the mentioned rules